src/lib/components/compositions/dialog/dialog.svelte

```diff
@@ -41,23 +44,54 @@
 </script>
 
 <!-- Inspired by https://github.com/mvolkmann/svelte-dialog -->
-<!-- TODO: Animate modal from bottom and fade in -->
 <dialog
   id={dialogId}
   bind:this={dialogEl}
-  class={`tw_dialog-box dark:tw_dialog-box_dark md:tw_dialog-box_medium`}
+  class={`tw_dialog-box dark:tw_dialog-box_dark md:tw_dialog-box_medium ${
+    forceOpen ? '' : 'tw_dialog-box_animate'
+  }`}
+  open={forceOpen}
 >
-  <div class="tw_w-full tw_-mt-4 tw_relative tw_text-right">
-    <button
-      class="tw_dialog-x tw_focusable-element dark:tw_focusable-element_dark"
-      on:click={() => closeDialog({ reason: 'user' })}
-      aria-controls={dialogId}
-    >
-      <XIcon
-        classes="tw_inline-block tw_fill-current tw_text-secondary-dark dark:tw_text-secondary-light"
-        ><T key="closeModal" /></XIcon
+  {#if withHeader}
```

Comment:
Because the order of HTML is different, and styles are quite different, it was just easier to split the HTML by a simple condition.

Reply:
Makes sense